ipn/ipnlocal/local.go

```diff
@@ -6788,6 +6838,14 @@ func longLatDistance(fromLat, fromLong, toLat, toLong float64) float64 {
 	return earthRadiusMeters * c
 }
 
+// shouldAutoExitNode checks for the auto exit node MDM policy.
+func shouldAutoExitNode() bool {
+	if exitNodeIDStr, _ := syspolicy.GetString(syspolicy.ExitNodeID, ""); exitNodeIDStr == "auto" {
```

Comment:
Late-breaking thoughts:

1. Move the magic string into a constant so it can be shared between here and `setExitNodeID`.
2. Let's call it `"auto:any"` to start, to allow later expansion into other selection algorithms.